infrastructure/module/eks/sa.helm_release.tf

```diff
@@ -35,4 +35,14 @@ resource "helm_release" "alb-controller" {
     name  = "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
     value = module.lb_role.iam_role_arn
   }
+
```

Comment:
You don't need this at all.

Reply:
Have we achieved that by applying `attach_load_balancer_controller_policy = true` in the Load balancer role creation file `sa.lb_role.tf`?

Reply:
No, this is webhook for mutation of service type and we will not utilise that feature, check docs.

Reply:
Just to clarify, I am talking about this added stuff, not about: **serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn**